render.yaml

```diff
@@ -121,16 +126,16 @@ services:
     buildCommand: cd backend && npm clean-install && npm run build
     startCommand: cd backend && npm run start
     envVars:
-      - key: NEAR_EXPLORER_WAMP_HOST
+      - key: NEAR_EXPLORER_CONFIG__WAMP__HOST
         fromService:
           type: web
           name: wamp
           property: host
-      - key: NEAR_EXPLORER_WAMP_BACKEND_SECRET
+      - key: NEAR_EXPLORER_CONFIG__WAMP__BACKEND_SECRET
         fromService:
           type: web
           name: wamp
-          envVarKey: NEAR_EXPLORER_WAMP_BACKEND_SECRET
+          envVarKey: NEAR_EXPLORER_CONFIG__BACKEND_SECRET
```

Comment:
1. Don't forget to update the secrets env names on Render
2. FYI, You will need to manually remove the unused variable names as Render does not remove those automatically

Reply:
Sure.
I'll make another PR to migrate DB env variables to this system as well, because it requires a little more effort